deploy/kiali/kiali_cr.yaml

```diff
@@ -53,6 +53,7 @@ spec:
 #    prometheus: prom-ns
 # means Prometheus is installed in the namespace "prom-ns" but all other
 # Istio components are installed in the namespace defined in istio_namespace.
+# NOTE: You can find Istio Components Status settings under `external_services`. This is only used by the operator.
```

Comment:
cc @jshaughn can you confirm this part? I actually thought some code you wrote in the server also uses this `istio_component_namespaces` setting. But I am not familiar. So I will add you as a reviewer to check this. It could just mean we remove the "This is only used by the operator." part - that's the part that I am not sure is true.

Reply:
@jmazzitelli you are right. It's been used in the kiali server. I misread the comment in the discussion. Removing this.

Reply:
Leaving only the first part: 

> NOTE: You can find Istio Components Status settings under `external_services`

Reply:
I'm not exactly sure but I think there is code in the graph-gen that wants to know if the selected namespace is an "istio" namespace because then, presumably, we shouldn't expect them to have sidecars, or something like that.  We should revisit this because it may date back to Mixer.